OpenRA.Mods.D2k/OpenRA.Mods.D2k.csproj

```diff
@@ -68,14 +68,19 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="Activities\CarryUnit.cs" />
+    <Compile Include="Activities\PickupUnit.cs" />
+    <Compile Include="Activities\DeliverUnit.cs" />
     <Compile Include="Activities\SwallowActor.cs" />
     <Compile Include="SpriteLoaders\R8Loader.cs" />
     <Compile Include="Traits\AttackSwallow.cs" />
-    <Compile Include="Traits\AutoCarryall.cs" />
+    <Compile Include="Traits\Carryall.cs" />
+    <Compile Include="Traits\Buildings\FreeActorWithDelivery.cs" />
+    <Compile Include="Traits\Buildings\ProductionFromMapEdge.cs" />
     <Compile Include="Traits\Buildings\DamagedWithoutFoundation.cs" />
     <Compile Include="Traits\Buildings\LaysTerrain.cs" />
     <Compile Include="Traits\Carryable.cs" />
+    <Compile Include="Traits\Player\HarvesterInsurance.cs" />
+    <Compile Include="Traits\Render\WithBuildingPlacedOverlay.cs" />
```

Comment:
No, this is the real error:

`CSC : error CS2001: Source file 'Traits\Render\WithBuildingPlacedOverlay.cs' could not be found`

not the warnings in the test project.